src/config.c

```diff
@@ -554,6 +553,13 @@ bool load_capture_config(const char *filename, struct capture_conf *config) {
   os_strlcpy(config->filter, value, MAX_FILTER_SIZE);
   os_free(value);
 
+  // Load middleware params
+  value = os_zalloc(INI_BUFFERSIZE);
+  ini_gets("capture", "middlewareParams", "", value, INI_BUFFERSIZE, filename);
+
+  os_strlcpy(config->middleware_params, value, MAX_MIDDLEWARE_PARAMS_SIZE);
+  os_free(value);
```

Comment:
Instead of allocating on the heap, (and having to check whether `os_zalloc()` fails), we could just allocated a buffer on the stack instead, which will make memory management so much easier.

```suggestion
  char ini_buffer[INI_BUFFERSIZE] = "";
  ini_gets("capture", "middlewareParams", "", ini_buffer, INI_BUFFERSIZE, filename);

  os_strlcpy(config->middleware_params, ini_buffer, MAX_MIDDLEWARE_PARAMS_SIZE);
```